ignite/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -131,12 +140,24 @@ func (a Account) Address(accPrefix string) string {
 		accPrefix = AccountPrefixCosmos
 	}
 
-	return toBench32(accPrefix, a.Info.GetPubKey().Address())
+	// TODO: handle error
```

Comment:
@ilgooz do you plan to address those TODOs in a later PR ?

Reply:
Yes, #2679 tracks this

Reply:
:pray: